addons/advanced_fatigue/functions/fnc_mainLoop.sqf

```diff
@@ -23,6 +23,8 @@ if (!alive ACE_player) exitWith {
     _staminaBarContainer ctrlCommit 1;
 };
 
+private _oxygen = (ACE_player getVariable [QEGVAR(medical,spo2), 90]) / 100;
```

Comment:
should default be 
`DEFAULT_SPO2 97`
or maybe 100 so that if medical isn't loaded this won't have any effect?

Reply:
AF uses 0.9 as a constant for oxygen saturation

Reply:
Addressed in latest commit